geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AttributeTableV5.scala

```diff
@@ -35,7 +35,7 @@ object AttributeTableV5 extends GeoMesaTable with LazyLogging {
   override def supports(sft: SimpleFeatureType) =
     sft.getSchemaVersion < 6 && SimpleFeatureTypes.getSecondaryIndexedAttributes(sft).nonEmpty
 
-  override val suffix: String = "attr_idx"
+  override val suffix: String = "attr"
```

Comment:
you can't, but it would screw up the query planning if we didn't change this i think...